Breeding-Insight/bi-api
# BI-1965 - Upgrade Gigwa to v2.6.2

# Description
**Story:** https://breedinginsight.atlassian.net/browse/BI-1965

- Fixed HTTP header from `X-FORWARDED-SERVER` (used in previous Gigwa version) to `X-FORWARDED-FOR` (v2.6.2)
- bumped mongo version



# Dependencies
gigwa-pom/feature/BI-1965

# Testing
- Build the gigwa docker image locally, then run the `GigwaGenotypeServiceImplIntegrationTest` and ensure it passes
- Run DeltaBreed, and ensure you can upload a VCF file successfully


# Checklist:
